frontend/packages/console-dynamic-plugin-sdk/src/app/__tests__/AppInitSDK.spec.tsx

```diff
@@ -90,4 +91,21 @@ describe('AppInitSDK', () => {
     expect(mockConfig.apiDiscovery).toHaveBeenCalledTimes(1);
     expect(mockConfig.apiDiscovery).toHaveBeenCalledWith(store);
```

Comment:
should we spy on initApiDiscoverySpy again here and ensure it is not called when a store instance is provided to cover all bases?

Reply:
updated.